package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
-  "version": "2.6.14",
+  "version": "2.6.15",
```

Comment:
You do not need to change this version atm as there is no release associated with current `2.6.14` version,